bot/exts/evergreen/emoji_count.py

```diff
@@ -17,73 +19,71 @@ class EmojiCount(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    def embed_builder(self, emoji: dict) -> discord.Embed:
+    @staticmethod
+    def embed_builder(emoji: dict) -> Tuple[discord.Embed, str]:
         """Generates an embed with the emoji names and count."""
         embed = discord.Embed(
             color=Colours.orange,
             title="Emoji Count",
             timestamp=datetime.datetime.utcnow()
         )
+        msg = []
 
         if len(emoji) == 1:
-            for key, value in emoji.items():
-                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
-                embed.set_thumbnail(url=random.choice(value).url)
+            for category_name, category_emojis in emoji.items():
+                msg.append(f"There is **{len(category_emojis)}** emoji in the **{category_name}** category")
+                embed.set_thumbnail(url=random.choice(category_emojis).url)
+
         else:
-            msg = ''
-            for key, value in emoji.items():
-                emoji_choice = random.choice(value)
-                emoji_info = f'There are **{len(value)}** emojis in the **{key}** category\n'
-                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
-            embed.description = msg
-        return embed
+            for category_name, category_emojis in emoji.items():
+                emoji_choice = random.choice(category_emojis)
+                emoji_info = f'There are **{len(category_emojis)}** emojis in the **{category_name}** category\n'
```

Comment:
You could actually remove the `\n` here.
Here are a few screenshots:
**With \n**
![image](https://user-images.githubusercontent.com/37217450/95546824-2ec72880-0a1f-11eb-940d-5ef8cf1239ee.png)

**Without \n**
![image](https://user-images.githubusercontent.com/37217450/95546843-3686cd00-0a1f-11eb-8a32-27ac6d4ef29b.png)

Experiment with it and see which one looks better. I'm leaning towards removing the `\n`, but do check with more emoji categories.

Reply:
Actually I am using linepaginator that's y it's automatically add spaces in main pydis server we have many category that's y i am using linepaginator so it don't get errored out